aequilibrae/paths/results/path_results.py

```diff
@@ -140,19 +145,24 @@ def reset(self) -> None:
         else:
             raise ValueError("Exception: Path results object was not yet prepared/initialized")
 
-    def update_trace(self, destination: int) -> None:
+    def update_trace(self, destination: int, early_exit: bool = False) -> None:
```

Comment:
I would not add the early_exit here and just use the same in the path_computation.  This is certainly an extra feature, but I am not sure it would ever be used and it may just create confusion.

Reply:
I've removed this arg and added a private variable to track the status of the tree instead. If they want to affect the behaviour of the `compute_path` call within `update_trace`, the public var `PathResults.early_exit` can be set .